### DISCUSSION TITLE
Lab5 Submission

### DISCUSSION THREAD

**Discussion by Russ420** [bol-edu/HLS-SOC-Discussions#145] [Q&A] [answered]
請問 Lab5 需要繳交什麼檔案與檔案的形式為何？

**Comment by Andy-Ko-0620** [comment, marked as answer]
同學你好
助教三天前在labi的repo有更新lab5-caravel FPGA.pdf的資訊
連結在: [https://github.com/bol-edu/caravel-soc_fpga-lab/tree/main/labi](https://github.com/bol-edu/caravel-soc_fpga-lab/tree/main/labi)

**Reply by Russ420** [reply]
謝謝你的回覆！
